kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -3258,6 +3258,14 @@ object KyuubiConf {
       .booleanConf
       .createWithDefault(true)
 
+  val ENGINE_SPARK_NOTEBOOK_MODE: ConfigEntry[Boolean] =
+    buildConf("kyuubi.engine.spark.notebook.mode")
+      .doc("Enable this to support notebook well." +
+        " For python notebook, the `text/plain` wrapped output will be returned.")
```

Comment:
I'm not quite familiar with this area, could you please elaborate more about the difference between this configuration true/false?

Reply:
from the logic of the code, maybe we can name it as `kyuubi.engine.spark.output.mode` with options "auto" and "notebook".

cc @cfmcgrady do you have suggestions?

Reply:
for notebook mode, for the output:
```
{ "text/plain": <content>}
```

for non-notebook mode, the output:
```
<content>
```